### PR TITLE
CC-12684: Minimal avalara integration.

### PR DESCRIPTION
**Pull request by AsonUnique**
- Developer(s): @AsonUnique

- Ticket: https://spryker.atlassian.net/browse/CC-12864

- Academy PR: ACADEMY_URL_HERE

- Release Group: https://release.spryker.com/release-groups/view/3457

- merge: merge

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   AvalaraTaxExtension   | minor (currently 0.0.x)  |                       |

-----------------------------------------

##### Change log

Initial Release

- The module provides plugin interfaces for extending the `AvalaraTax` module.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          composer --version
+          composer install
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: mbstring, intl, pdo_mysql
+          tools: composer:v2
+
+#       suite-tests:
```
Comment (abitskil) [inline .github/workflows/ci.yml]: * Remove commented part.

Comment (abitskil) [reply .github/workflows/ci.yml]: Resolved.

**Review by abitskil — APPROVED** [review]

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+indent_size = 4
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.bat]
+end_of_line = crlf
+
+[composer.json]
+indent_size = 2
```
Comment (dereuromark) [inline .editorconfig:16]: This is invalid/wrong, see master

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,66 @@
+name: CI
+
+on:
+  pull_request:
```
Comment (dereuromark) [inline .github/workflows/ci.yml:4]: wrong indentation, see master

File: README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# AvalaraTaxExtension Module
+[![Build Status](https://travis-ci.org/spryker-eco/avalara-tax-extension.svg)](https://travis-ci.org/spryker-eco/avalara-tax-extension)
+[![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%207.2-8892BF.svg)](https://php.net/)
```
Comment (dereuromark) [inline README.md:3]: 7.3

**Pull request merged**